.license-headers/kotlin.txt

```diff
@@ -0,0 +1,3 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why do Kotlin files use an abbreviated license header?

Reply:
I'm not sure exactly, I just formalized what was already in the files before my changes.